Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM eclipse-temurin:21-jdk-alpine
+
+WORKDIR /app
+
+COPY .mvn/ .mvn
+COPY mvnw pom.xml ./
+RUN ./mvnw dependency:resolve
+
+COPY src ./src
+
+CMD ["./mvnw", "spring-boot:run"]
```

Comment:
**Suggestion:** Consider adding a .dockerignore file to prevent unnecessary files from being copied into the Docker image. This can help to reduce the size of the image and speed up the build process. [enhancement]
```suggestion
# No changes in Dockerfile, but add a .dockerignore file with the following content:
.git
.gitignore
Dockerfile
README.md
target/
```